### PR TITLE
fix security issue

### PR DESCRIPTION
**Pull request by asockalingam**

### CODE REVIEW

File: db-async-common/src/main/java/com/github/jasync/sql/db/ConnectionPoolConfiguration.kt [hunk 1]
```diff
@@ -118,6 +118,16 @@ data class ConnectionPoolConfiguration @JvmOverloads constructor(
         coroutineDispatcher = coroutineDispatcher
     )
 
+    override fun toString() = "ConnectionPoolConfiguration(host=$host, port=REDACTED, database=$database, " +
+            "username=REDACTED, password=REDACTED, maxActiveConnections=$maxActiveConnections, " +
+            "maxIdleTime=$maxIdleTime, maxPendingQueries=$maxPendingQueries, " +
+            "connectionValidationInterval=$connectionValidationInterval, " +
+            "connectionCreateTimeout=$connectionCreateTimeout, connectionTestTimeout=$connectionTestTimeout,\n" +
+            "queryTimeout=$queryTimeout, eventLoopGroup=$eventLoopGroup, " +
```
Comment (x0a1b) [inline db-async-common/src/main/java/com/github/jasync/sql/db/ConnectionPoolConfiguration.kt:126]: Why do we need to stringify eventLoopGroup

File: db-async-common/src/main/java/com/github/jasync/sql/db/ConnectionPoolConfiguration.kt [hunk 2]
```diff
@@ -118,6 +118,16 @@ data class ConnectionPoolConfiguration @JvmOverloads constructor(
...
+            "executionContext=$executionContext, coroutineDispatcher=$coroutineDispatcher, ssl=$ssl\n" +
```
Comment (x0a1b) [inline db-async-common/src/main/java/com/github/jasync/sql/db/ConnectionPoolConfiguration.kt:127]: Same is true for `executionContext` and `coroutineDispatcher`

File: db-async-common/src/main/java/com/github/jasync/sql/db/ConnectionPoolConfiguration.kt [hunk 3]
```diff
@@ -118,6 +118,16 @@ data class ConnectionPoolConfiguration @JvmOverloads constructor(
         coroutineDispatcher = coroutineDispatcher
     )
 
+    override fun toString() = "ConnectionPoolConfiguration(host=$host, port=REDACTED, database=$database, " +
```
Comment (x0a1b) [inline db-async-common/src/main/java/com/github/jasync/sql/db/ConnectionPoolConfiguration.kt:121]: You can use multiline string feature here

```kotlin
"""
...
"""
```

File: db-async-common/src/main/java/com/github/jasync/sql/db/ConnectionPoolConfiguration.kt (hunk 1 above)
Comment (asockalingam) [reply db-async-common/src/main/java/com/github/jasync/sql/db/ConnectionPoolConfiguration.kt]: fixed.

File: db-async-common/src/main/java/com/github/jasync/sql/db/ConnectionPoolConfiguration.kt (hunk 2 above)
Comment (asockalingam) [reply db-async-common/src/main/java/com/github/jasync/sql/db/ConnectionPoolConfiguration.kt]: fixed.

**Pull request merged**